src/cluster_asm.h

```diff
@@ -43,9 +43,11 @@ sds asmCatInfoString(sds info);
 void clusterMigrationCommand(client *c);
 void clusterSyncSlotsCommand(client *c);
 struct asmTask *asmLookupTaskBySlotRangeArray(struct slotRangeArray *sra);
-void asmTrimSlotsIfNotOwned(void);
 void asmCancelTrimJobs(void);
-
+void asmNotifySlavesStateOnFullSync(void);
+void asmHandleOnChangeMaster(void);
+void asmHandleOnPromoteToMaster(void);
```

Comment:
```suggestion
void asmHandleMasterChange(void);
void asmHandlePromotionToMaster(void);
```

perhaps we should either use `handle` or `on` in the name, it feels more natural this way.

Reply:
make sense, thank you